templates/common/infra/terraform/core/database/postgresql/postgresql_output.tf

```diff
@@ -11,6 +11,10 @@ output "AZURE_POSTGRESQL_SPRING_DATASOURCE_URL" {
   value = "jdbc:postgresql://${azurerm_postgresql_flexible_server.psql_server.fqdn}:5432/${azurerm_postgresql_flexible_server_database.database.name}?sslmode=require"
 }
 
-output "AZURE_POSTGRESQL_ADMIN_USERNAME" {
-  value = azurerm_postgresql_flexible_server_active_directory_administrator.aad_admin.principal_name
+output "AZURE_POSTGRESQL_USERNAME" {
+  value = local.psqlUserName
+}
+
+output "AZURE_POSTGRESQL_PASSWORD" {
+  value = random_password.password[1].result
```

Comment:
We should mark this as `sensitive = true`: https://developer.hashicorp.com/terraform/tutorials/configuration-language/outputs#redact-sensitive-outputs